api/src/main/java/com/datastrato/gravitino/authorization/SecurableObject.java

```diff
@@ -4,21 +4,24 @@
  */
 package com.datastrato.gravitino.authorization;
 
-import com.datastrato.gravitino.annotation.Evolving;
+import com.datastrato.gravitino.annotation.Unstable;
 import javax.annotation.Nullable;
 
 /**
  * The securable object is the entity which access can be granted. Unless allowed by a grant, access
  * is denied. Gravitino organizes the securable objects using tree structure. The securable object
- * may be a catalog, a table or a schema, etc. For example, `catalog1.schema1.table1` represents a
- * table named `table1`. It's in the schema named `schema1`. The schema is in the catalog named
- * `catalog1`. Similarly, `catalog1.schema1.topic1` can represent a topic.
- * `catalog1.schema1.fileset1` can represent a fileset. `*` represents all the catalogs. If you want
- * to use other securable objects which represents all entities," you can use their parent entity,
+ * may be a catalog, a table or a schema, etc. <br>
+ * For example, `catalog1.schema1.table1` represents a table named `table1`. It's in the schema
+ * named `schema1`. The schema is in the catalog named `catalog1`. <br>
+ * Similarly, `catalog1.schema1.topic1` can represent a topic.`catalog1.schema1.fileset1` can
+ * represent a fileset.<br>
+ * `*` can represent all th metalakes <br>
+ * `metalake` can represent all the catalogs of this metalake. <br>
+ * To use other securable objects which represents all entities," you can use their parent entity,
```

Comment:
I would suggest you add more descriptions and examples about this securable object.

Reply:
I refactored the comment.